rust/telemetry/src/lib.rs

```diff
@@ -66,7 +66,7 @@ impl Telemetry {
                 environment: Some(environment.into()),
                 // We can't get the release number ourselves because we don't know if we're embedded in a GUI Client or a Headless Client.
                 release: Some(release.into()),
-                traces_sample_rate: 1.0,
+                traces_sample_rate: 0.1,
```

Comment:
We'll still get all errors and messages, this is for just (lowercase 't') tracing, right?

Reply:
Yeah I think this is for sending the "transactions" as Sentry calls them. The docs recommend to set this to < 1.0. Maybe 10% is even too much? Perhaps 1% is better?

Reply:
10% is probably fine, I only trigger transactions for lifecycle events like starting connlib, so it won't happen often per system

Reply:
I'll have to double check but I think tracing spans get picked up too. We don't have any on WARN or above though so it should be fine.

Reply:
So yeah, all tracing spans of level INFO and above are captured as transactions.